Tools/Frame_params/deset-mapping-boat.param

```diff
@@ -0,0 +1,59 @@
+#NOTE: JapanDrones DeSET mapping boat parameters for ArduPilor Rover-4.2 (or higher). Serial1:Herelink receiver,  Serial2:Torqeedo motor adapter, Serial3:dAISy2+, Serial4:NMEA Sonar
+ACRO_TURN_RATE,60
+AIS_TYPE,1
+ATC_ACCEL_MAX,0.3
+ATC_BRAKE,0
+ATC_SPEED_FLTE,5
+ATC_SPEED_I,0.2
+ATC_SPEED_IMAX,0.5
+ATC_SPEED_P,0.2
+ATC_STR_ACC_MAX,120
+ATC_STR_ANG_P,2
+ATC_STR_RAT_FF,0.4
+ATC_STR_RAT_FLTE,5
+ATC_STR_RAT_I,0.2
+ATC_STR_RAT_MAX,90
+ATC_STR_RAT_P,0.15
+ATC_TURN_MAX_G,0.2
+BATT_MONITOR,23
+BRD_SAFETYENABLE,0
+CRUISE_SPEED,3.1
+CRUISE_THROTTLE,100
+FRAME_CLASS,2
+FRAME_TYPE,0
+MOT_SLEWRATE,25
+MOT_VEC_ANGLEMAX,30
+NAVL1_DAMPING,0.8
+NAVL1_PERIOD,16
+RNGFND1_MAX_CM,20000
+RNGFND1_MIN_CM,0
+RNGFND1_TYPE,17
+SERIAL2_BAUD,19
+SERIAL2_PROTOCOL,39
+SERIAL3_BAUD,38
+SERIAL3_PROTOCOL,40
+SERIAL4_BAUD,9
+SERIAL4_PROTOCOL,9
+SERVO_BLH_3DMASK,0
+SERVO_BLH_AUTO,0

```

Comment:
A lot of these are set to their defaults?

Reply:
nice catch thanks!  that was totally a mistake and I've removed all these param now.